app/main/routes.py

```diff
@@ -636,6 +639,11 @@ def record(record_id: uuid.UUID):
 @access_token_sign_in_required
 def download_record(record_id: uuid.UUID):
     file = db.session.get(File, record_id)
+    ayr_user = AYRUser(session.get("user_groups"))
+    can_download_records = ayr_user.can_download_records
+
+    if can_download_records is False:
+        abort(401)
```

Comment:
I think technically this should be a 403 as the user is authorized but just is forbidden from this.

Reply:
yup I agree should be 403, will change